.github/workflows/pull-request.yml

```
@@ -0,0 +1,36 @@
+name: Validate Pull Request
+
+on: [push, pull_request]
```

We can remove push here. Otherwise this may cause duplicate runs for each PR.

```suggestion
on: [pull_request]
```

---

Moved to dev build workflow in f5fbdd3c1a06775fa66ec9b2013be925a0e0cbe0 as discussed in https://github.com/ballerina-platform/module-ballerinax-prometheus/pull/1#discussion_r557854204